package.json

```diff
@@ -4,15 +4,21 @@
   "description": "",
   "main": "index.js",
   "scripts": {
+    "build": "rm -rf dist && ttsc && next build",
     "dev:server": "cross-env NODE_ENV=development npm run ts:exec server/index.ts",
     "dev": "nodemon",
     "docker:db": "docker-compose exec pg psql -U postgres -d dev-directory",
     "docker:deps": "docker-compose up",
+    "jest": "cross-env NODE_ENV=test npx jest --runInBand",
     "lint:fix": "eslint . --fix",
     "lint": "eslint .",
-    "test:server": "cross-env NODE_ENV=test npx jest --runInBand ./server",
-    "test:watch": "cross-env NODE_ENV=test npx jest --watch",
-    "test": "cross-env NODE_ENV=test npx jest",
+    "migrate": "npm run sequelize-cli db:migrate",
+    "migrate:undo": "npm run sequelize-cli db:migrate:undo",
+    "sequelize-cli": "cross-env NODE_ENV=development sequelize-cli",
+    "start": "cross-env NODE_ENV=production node dist/server/index",
+    "test:server": "npm run jest -- ./server",
+    "test:watch": "npm run jest -- --watch",
+    "test": "npm run jest",
```

Comment:
All tests should have the `--runInBand` flag; this (along with the `jest` script above) makes it so that we don't have to repeat it